apps/happy-blocks/index.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Plugin Name: Happy Blocks
+ * Description: Happyness Engineering Specific Blocks
+ * Author: A8C
+ * Text Domain: full-site-editing
```

Comment:
We definitely need to change the textdomain to something unique.

Reply:
AFAIK every plugin declares and uses its own textdomain, and that applies to dotcom's plugins as well.

Let's keep the unique one you already added, and we can always alter it to something common if we find a reason to.